packages/lesswrong/components/localGroups/LocalGroupsList.tsx

```diff
@@ -18,24 +18,40 @@ const styles = createStyles((theme: ThemeType): JssStyles => ({
   }
 }))
 
-const LocalGroupsList = ({terms, children, classes, showNoResults=true}: {
+const LocalGroupsList = ({terms, children, classes, showNoResults=true, heading}: {
   terms: LocalgroupsViewTerms,
   children?: React.ReactNode,
   classes: ClassesType,
   showNoResults?: boolean,
+  heading?: string,
 }) => {
   const { results, count, loadMore, totalCount, loading, loadingMore, loadMoreProps } = useMulti({
     terms,
     collectionName: "Localgroups",
     fragmentName: 'localGroupsHomeFragment',
     enableTotal: false,
   });
-  const { LocalGroupsItem, Loading, PostsNoResults, SectionFooter, LoadMore } = Components
+  const { LocalGroupsItem, Loading, PostsNoResults, SectionFooter, LoadMore, SingleColumnSection, SectionTitle } = Components
+
+  const MaybeTitleWrapper = ({children}) => heading ?
+    <SingleColumnSection>
+      <SectionTitle title={heading} />
+      {children}
+    </SingleColumnSection> :
+    children;
 
   if (!results && loading) return <Loading />
-  if ((results && !results.length) && showNoResults) return <PostsNoResults />
 
-  return <div>
+  // if we are given a section title/heading,
+  // then we can make sure to hide it when showNoResults is false and there are no results to show
+  if (!results || !results.length) {
+    return showNoResults ? 
+      <MaybeTitleWrapper><PostsNoResults /></MaybeTitleWrapper> :
+      null
+  }
```

Comment:
Jonathan and I added this so that the new section would not appear for a user with no groups (which is most users)